strings/tcg/set_one.json

```
@@ -766,44 +766,44 @@
 			"id": "doc",
 			"name": "Medical Doctor",
 			"desc": "Nanotrasen's doctors are well known for their ability to treat almost any ailment known to mankind... as well as causing a fair few in the process.",
-			"rules": "You may tap this card: Select a card that has less attack than this card from your graveyard, and summon it to your side of the field.",
+			"rules": "Tap: Add a card from your discard pile to your hand, except another 'Medical Doctor'.",
 			"icon_state": "md",
-			"power": "2",
+			"power": "1",
 			"resolve": "3",
 			"faction": "Medical",
 			"summoncost": "3",
 			"cardtype": "Creature",
-			"cardsubtype": "Human Doctor",
+			"cardsubtype": "Medical",
 			"rarity": "common",
 			"summon_icon_state": "Medical Doctor"
 		},
 		{
 			"id": "mime",
 			"name": "Mime",
 			"desc": "Si vous regardez attentivement dans les yeux d'un mime, vous pouvez voir le tourment sans fin derrière leur façade silencieuse. C'est vraiment tragique.",
-			"rules": "You may tap this card: Pick an opponent's card and nullify its effect until it leaves play.",
+			"rules": "Tap: Target creature and all its attached equipment loses all its effect text until this card leaves the field.",
 			"icon_state": "mime",
-			"power": "2",
+			"power": "0",
 			"resolve": "1",
 			"faction": "Service",
 			"summoncost": "1",
 			"cardtype": "Creature",
-			"cardsubtype": "Mime",
+			"cardsubtype": "Bar",
 			"rarity": "uncommon",
 			"summon_icon_state": "Mime"
 		},
 		{
 			"id": "miningborg",
 			"name": "Cyborg (Mining Shell)",
 			"desc": "Fitted with a drill and tracks, the Mining Shell is designed to hold up to the rigors of mining, be that on the hellish surface of Indecipheres, or in the silent vacuum of the asteroid belt.",
-			"rules": "{$Asimov}, at the end of your turn, if this card is not tapped, you may tap this card at the start of your next turn to gain 1 mana.",
+			"rules": "{$Asimov} At the start of your turn: Tap: Gain 2 mana.",
```

```suggestion
			"rules": "{$Asimov} At the start of your turn: Tap: Gain 2 plasma.",
```
Additionally, based on the cost, is this a permanent gain of plasma, ala hearthstone's Wild Growth, or a single turn where they get extra temporary mana for a single turn?

---

Just for this turn, temporary mana.